wp-content/themes/pub/wporg-learn-2020/css/components/_search-form.scss

```diff
@@ -0,0 +1,126 @@
+
+/*--------------------------------------------------------------
+# .search-form
+--------------------------------------------------------------*/
+.search-form {
+	font-size: 0;
+	margin-bottom: 2rem;
+	max-width: 100%;
+	position: relative;
+}
+
+.search-form .search-field {
+	border: none;
+	border-radius: 0;
+	box-shadow: none;
+	display: block;
+	font-size: 1rem;
+	margin: 0 auto;
+	max-width: 100%;
+	padding: 0.5rem;
+	width: 22.7373675443rem;
+}
+
+.search-form .button-search {
+	border-top: 0;
+	border-left: none;
+	border-radius: 0 2px 2px 0;
+	font-size: 12px;
+	position: relative;
+	right: auto;
+	top: auto;
+	vertical-align: top;
+}
+
+.search-form .button-search:active {
+	background: #006799;
+	border-right: 1px solid #006799;
+	box-shadow: none;
+}
+
+.search-form .button-search .dashicons {
+	font-size: 1rem;
+}
+
+.site-header:not(.home) .search-form .button-search .dashicons {
+	font-size: 16px;
+	line-height: 20px;
+}
+
+.site-header.home form {
+	display: inline-block;
+}
+
+.site-header.home .search-form .button-search,
+.site-main .search-form .button-search {
+	background: transparent;
+	border: none;
+	border-radius: 0;
+	box-shadow: none;
+	color: #32373c;
+	display: block;
+	height: 45px;
+	padding: 0.5rem 1rem;
+	position: absolute;
+	right: 0;
+	text-shadow: none;
+	top: 0;
+}
+
+.site-header.home .search-form .button-search:focus,
+.site-main .search-form .button-search:focus {
+	box-shadow: 0 0 2px 1px #33b3db;
+}
+
+.site-header.home .search-form .button-search:active,
+.site-main .search-form .button-search:active {
+	background: transparent;
+	border: none;
+	transform: none;
+}
+
+.site-header:not(.home) .search-form {
+	margin: 0;
+}
+
+.site-header:not(.home) .search-form .search-field {
+	border: 0;
+	border-radius: 2px 0 0 2px;
+	display: inline-block;
+	font-size: 1rem;
+	padding: 5px 10px;
+	position: relative;
+}
+
+@media screen and (min-width: 48em) {
```

Comment:
I'm seeing an awkward search button size at 768px wide — maybe you should use the `$breakpoint-tablet` var, or `breakpoint` mixin here?

![Screen Shot 2020-08-31 at 2 31 57 PM](https://user-images.githubusercontent.com/541093/91754385-506aff80-eb97-11ea-9e60-5377568bf099.png)

Also the download button flashes visible at just this side, which is odd, but I think it comes from the legacy styles. No change requested here, just a 🤨

Reply:
Yeah, I remember a meta ticket about this. I wonder if it has been fixed, and/or the copied CSS for this theme was before that fix.